puppet/zulip/manifests/app_frontend_base.pp

```diff
@@ -5,11 +5,14 @@
   include zulip::nginx
   include zulip::supervisor
 
-  $web_packages = [
-    # Needed to access our database
-    "postgresql-client-${zulip::base::postgres_version}",
-  ]
-  zulip::safepackage { $web_packages: ensure => 'installed' }
+  if $::osfamily == 'debian' {
+    # This is not necessary on CentOS because $postgresql package already includes the client
```

Comment:
Or rather, `$postgresql` is client, `"${postgresql}-server"` is server.

Reply:
This isn't duplicate -- if you split the Zulip service across multiple systems, then this role just needs the client, not also the server.

Reply:
If I didn't disable this on CentOS, puppet would have complained that postgres_common.pp has already enforced the existence of `postgresql10`. I can add this client-only requirement only if I specifically remove the `postgresql10` requirement from postgres_common.pp.

Reply:
I think that just means we should declare a dependency on `postgresql` here, and `postgresql-server` in `postgres_common.pp`?